modules/packages/manifests/xcode_cmd_line_tools.pp

```diff
@@ -4,7 +4,7 @@
 
 class packages::xcode_cmd_line_tools {
 
-    packages::macos_package_from_s3 { 'Command_Line_Tools_macOS_10.14_for_Xcode_10.1.dmg':
+    packages::macos_package_from_s3 { 'Command_Line_Tools_for_Xcode_12.2.dmg':
```

Comment:
This might be big breaking change.  In the future (soon) each role will specify which package version it expects.  In the meantime, either base version on OS major version or test if the new xcode cli breaks on Mojave or Catalina.

Reply:
agreed. this is needed for the builders, but for testers I think keeping the old commandlinetools is safer